packages/ui/src/components/vuestic-mixins/ClickOutsideMixin/tests/ClickOutsideMixin.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { CreateElement } from 'vue'
+import { mount as vtuMount } from '@vue/test-utils'
```

Comment:
Maybe a word or two on why shallow mount is not enough?

Reply:
@asvae We can use the shallow mount here. My intention was to test some behavior with the child components, but it seems like overhead